src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -66,20 +71,21 @@ public Node(K key, V value, Node<K, V> next) {
     }
 
     private void resize() {
-        Node<K,V>[] tmpTable = table;
+        Node<K, V>[] tmpTable;
+        tmpTable = table;
```

Comment:
No need to make it in two lines, assign at once

Reply:
I have some problems with man package...so I had to do this
[VariableDeclarationUsageDistance]